website/content/en/docs/reference/configuration/_index.md

```diff
@@ -78,7 +78,7 @@ sources:
       - /var/log/apache2/*.log
     ignore_older: 86400
 transforms:
-  apache_parser:
+  remap:
```

Comment:
```suggestion
  apache_parser:
```
This is the ID, not the type!

Reply:
Ah, true, this needs to be fixed on `master` too. These changes were directly from https://github.com/vectordotdev/vector/commit/f262fea3cbf09ab8bfa1bcc4937964310d4b54e0

Reply:
Ah, actually a contributor fixed it in 96973fa24. I'll just cherry-pick that commit over too once this is merged.